docs/how-to-develop-scraper.md

```diff
@@ -154,72 +154,55 @@ If the website does not support Recipe Schema, or the schema does not include al
+The [BeautifulSoup documentation](https://www.crummy.com/software/BeautifulSoup/bs4/doc/index.html) is a good resource for getting started with extracting information from HTML. A guide of common patterns and best practice used in this library can be found [here](in-depth-guide-html-scraping).
 
 Some helper functions are available in the `_utils.py` file. These are functions that are commonly needed when extracting information from HTML, such as `normalize_string()`.
 
 ## 5. Create the test
 
-A test case was automatically created when the scraper class was created. It can be found in the `tests/` directory with the name `test_lowercaseclassname.py`. The HTML from the URL used to generate the scraper is also downloaded to the `tests/test_data/` directory.
-
-The generated test case will look something like this:
-
-```python
-from recipe_scrapers.newscraperclass import NewScraperClass
-from tests import ScraperTest
-
-class TestNewScraperScraper(ScraperTest):
-
-    scraper_class = NewScraperClass
-
-    def test_host(self):
-        self.assertEqual("websitehost.com", self.harvester_class.host())
-
-    def test_author(self):
-        self.assertEqual(None, self.harvester_class.author())
-
-    def test_title(self):
-        self.assertEqual(None, self.harvester_class.title())
-
-    def test_category(self):
-        self.assertEqual(None, self.harvester_class.category())
-
-    def test_total_time(self):
-        self.assertEqual(None, self.harvester_class.total_time())
-
-    def test_yields(self):
-        self.assertEqual(None, self.harvester_class.yields())
-
-    def test_image(self):
-        self.assertEqual(None, self.harvester_class.image())
-
-    def test_ingredients(self):
-        self.assertEqual(None, self.harvester_class.ingredients())
-
-    def test_instructions(self):
-        self.assertEqual(None, self.harvester_class.instructions())
-
-    def test_ratings(self):
-        self.assertEqual(None, self.harvester_class.ratings())
-
-    def test_cuisine(self):
-        self.assertEqual(None, self.harvester_class.cuisine())
-
-    def test_description(self):
-        self.assertEqual(None, self.harvester_class.description())
+A test case was automatically created when the scraper class was created. It can be found in the `tests/test_data/<host>` directory, where `host` is the hostname of the website the scraper is for.
+
+The test case comprises two parts:
+
+1. testhtml file containing the html from the URL used to generate the scraper
+2. json file containing the expected output from the scraper when the scraper is run on the testhtml file.
+
+The generated json file will look something like this, with only the host field populated:
+
+```json
+{
+  "author": "",
+  "canonical_url": "",
+  "host": "<host>",
```

Comment:
Do we need the `host` field in the test data?

Reply:
Fair point, probably not. 
I included it because I left the host field in when I converted all the tests. I don't think it does any harm to keep it in and the modifications I made to `generate.py` automatically populate it.